openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -60,10 +60,12 @@ static public void loadLibs() {
 
         if (!libsLoaded) {
             final CpuArchitecture cpuArchitecture = getCpuArchitecture(System.getProperty("os.arch"));
+            final boolean isAlpine = isAlpine();
```

Comment:
We aren't much concerned if we are using Alpine or other distribution. We are concerned if we are using musl or glibc. Do we have a way to check that instead of checking the distribution?

Reply:
Yes, you are right. We can use an env variable that indicates the libc implementation that must be used. I changed this code in the latest commit to use this alternative.